GetStream/stream-video-flutter
# [70] Implement CallParticipantsInfoView

### 🎯 Goal

Closes #70 

### 🎨 UI Changes

| Info | Invite |
| --- | --- |
| ![info](https://user-images.githubusercontent.com/1286516/210456367-d3c058dc-c141-419d-a7e7-54a774698b4c.png) | ![invite](https://user-images.githubusercontent.com/1286516/210456376-9d7131d2-f6cb-40ce-99c5-bcf08b57960e.png) |

### 🧪 Testing

- establish a call with `user A`
- join the same call with `user B`
- go to `CalParticipantsInfo` screen on `user A` side
- unmute `mic` on `user B` side (may take up to 5 seconds to take effect).
- observe on `user A` side that the `mic` icon turns into `unmuted` state.


### ☑️Contributor Checklist

#### General

### ☑️Reviewer Checklist
